qa/tasks/cephfs/fuse_mount.py

```diff
@@ -87,7 +87,8 @@ def _run_mount_cmd(self, mntopts, mntargs, check_status):
             stdin=self._mount_cmd_stdin,
             stdout=mountcmd_stdout,
             stderr=mountcmd_stderr,
-            wait=False
+            wait=False,
+            omit_sudo=False
```

Comment:
we shouldn't use sudo with FUSE right? Do we really need to omit_sudo then?

Reply:
Please read commit message for the explanation -

```
Although setting "omit_sudo" to "False" will have no impact on ceph-fuse
mount command during testing with teuthology (because teuthology code
does nothing whether omit_sudo is set or unset), let's set it to False
so that there's no chance for any bugs in future.
```